src/components/content/order/common/ScaleOrModifySubmitStatusAlert.tsx

```diff
@@ -0,0 +1,128 @@
+                    DeployedServiceDetails.serviceDeploymentState.MODIFICATION_SUCCESSFUL.toString() ||
+                deployedServiceDetails.serviceDeploymentState.toString() ===
+                    DeployedServiceDetails.serviceDeploymentState.MODIFICATION_FAILED.toString() ||
+                deployedServiceDetails.serviceDeploymentState.toString() ===
+                    DeployedServiceDetails.serviceDeploymentState.ROLLBACK_FAILED.toString()
+            ) {
+                return <ProcessingStatus response={deployedServiceDetails} operationType={OperationType.Modify} />;
+            }
+        } else if (isSubmitFailed) {
+            if (isSubmitFailed instanceof ApiError && isSubmitFailed.body && 'details' in isSubmitFailed.body) {
+                const response: Response = isSubmitFailed.body as Response;
+                return getOrderSubmissionFailedDisplay(response.details);
+            } else {
+                return getOrderSubmissionFailedDisplay([isSubmitFailed.message]);
+            }
+        } else if (uuid && isPollingError) {
+            return 'Modification status polling failed. Please visit MyServices page to check the status of the request.';
+        } else if (uuid) {
+            return 'Request accepted';
+        } else if (uuid === undefined) {
+            return 'Request submission in-progress';
+        }
+        return '';
+    }, [deployedServiceDetails, uuid, isPollingError, isSubmitFailed]);
+
+    const alertType = useMemo(() => {
+        if (isPollingError || isSubmitFailed) {
+            return 'error';
+        }
+        if (deployedServiceDetails) {
+            if (
+                deployedServiceDetails.serviceDeploymentState.toString() ===
+                    DeployedServiceDetails.serviceDeploymentState.MODIFICATION_FAILED.toString() ||
+                deployedServiceDetails.serviceDeploymentState.toString() ===
+                    DeployedServiceDetails.serviceDeploymentState.ROLLBACK_FAILED.toString()
+            ) {
+                return 'error';
+            }
+        }
+        return 'success';
+    }, [deployedServiceDetails, isPollingError, isSubmitFailed]);
+
+    if (isPollingError || isSubmitFailed) {
+        if (stopWatch.isRunning) {
+            stopWatch.pause();
+        }
+    }
+
+    if (deployedServiceDetails) {
+        if (
+            deployedServiceDetails.serviceDeploymentState.toString() ===
+                DeployedServiceDetails.serviceDeploymentState.MODIFICATION_FAILED.toString() ||
+            deployedServiceDetails.serviceDeploymentState.toString() ===
+                DeployedServiceDetails.serviceDeploymentState.ROLLBACK_FAILED.toString() ||
+            deployedServiceDetails.serviceDeploymentState.toString() ===
+                DeployedServiceDetails.serviceDeploymentState.MODIFICATION_SUCCESSFUL.toString()
+        ) {
+            if (stopWatch.isRunning) {
+                stopWatch.pause();
```

Comment:
stop watch is also reset when the modification is completed. It must not be reset.